config/routes.rb

```diff
@@ -8,6 +8,10 @@
   get "/admin/invoices", to: "admin/invoices#index"
   get "/admin/invoices/:invoice_id", to: "admin/invoices#show"
   get "/admin/merchants", to: "admin/merchants#index"
+  namespace :admin do
```

Comment:
Awesome. Still kind of new to these, but it's cool to see one in action. Eventually we can adapt all our routes to more advanced routing.